resource/sysrc_freebsd.go

```diff
@@ -0,0 +1,105 @@
+// +build freebsd
+
+package resource
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+)
+
+// SysRC is a resource which manages rc.conf variables.
+//
+// Example:
+//   rcvar = resource.sysrc.new("keyrate")
+//   rcvar.state = "present"
+//   rcvar.value = "fast"
+type SysRC struct {
+	Base
+	Value string `luar:"value"`
+}
+
+// NewSysRC creates a new resource for managing rc.conf variables
+// on a FreeBSD system.
+func NewSysRC(name string) (Resource, error) {
+	s := &SysRC{
+		Base: Base{
+			Name:          name,
+			Type:          "sysrc",
+			State:         "absent",
+			Require:       make([]string, 0),
+			PresentStates: []string{"present"},
+			AbsentStates:  []string{"absent"},
+			Concurrent:    false,
+		},
+	}
+
+	return s, nil
+}
+
+// Evaluate evaluates the state of the resource.
+func (s *SysRC) Evaluate() (State, error) {
+	state := State{
+		Current:  "unknown",
+		Want:     s.State,
+		Outdated: false,
+	}
+
+	out, err := exec.Command("sysrc", s.Name).CombinedOutput()
+	if err != nil {
+		state.Current = "absent"
+		state.Outdated = true
+		return state, nil
+	}
+	state.Current = "present"
+
+	k, v, err := parseSysRCOutput(string(out))
+	if err != nil {
+		return state, err
+	}
+
+	if s.Name != k {
+		return state, fmt.Errorf("bug: expected rcvar %v, got %v", s.Name, k)
+	}
+
+	if s.Value != v {
+		state.Outdated = true
+	}
+
+	return state, nil
+}
+
+// Create starts the service.
```

Comment:
Please update the doc comments for `Create`, `Delete` and `Update` methods, as the currently they suggest they are for services.